src/fittable_models.jl

```diff
@@ -0,0 +1,45 @@
+# Convenience function for wavelength conversion
+aa_to_invum(wave::Real) = 10000 / wave
+aa_to_invum(wave::Quantity) = aa_to_invum(ustrip(u"angstrom", wave))
+
+"""
+    FM90(C1 = 0.10, C2 = 0.70, C3 = 3.23, C4 = 0.41, xo = 4.60, γ = 0.9)(λ::Real)
+
+The parameters C1, C2, C3, C4, xo and γ are y-intercept, slope, bump amplitude, FUV rise amplitude,
+bump centroid and bump width respectively.
+
+Fitzpatrick & Massa (1990) 6 parameter ultraviolet shape model, this model is only applicable at UV wavelengths.
+
+If `λ` is a `Unitful.Quantity` it will be automatically converted to Å and the
+returned value will be `UnitfulAstro.mag`.
+"""
+struct FM90
+    C1::Real
+    C2::Real
+    C3::Real
+    C4::Real
+    xo::Real
+    γ::Real
```

Comment:
Please, pay attention to the types of the fields: they shouldn't be abstract types like `Real`.  Either use a fixed type (often too restrictive) or make the type parametric.